### PR TITLE
RHBPMS-4442 - Wrong number of executions shown in Job Request Details

### PR DESCRIPTION
**Pull request by mswiderski**
@nmirasch @cristianonicolai when you find some time...

I couldn't find a way to test these changes so all options/ideas for tests are more then welcome

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@mswiderski I also couldn't get a good way to test this without some refactor. I created another change to handle that, see https://github.com/droolsjbpm/jbpm-console-ng/pull/532. Adding test checks for the new column should be straightforward in there if you rebase your change.

**Comment by mswiderski** [comment]
@cristianonicolai rebased and improved slightly to test the added column, can you have a quick look?

File: jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-backend/src/main/java/org/jbpm/console/ng/es/backend/server/RequestSummaryHelper.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import org.jbpm.console.ng.es.model.RequestSummary;
+import org.kie.api.executor.ErrorInfo;
+import org.kie.api.executor.RequestInfo;
+import org.kie.api.executor.STATUS;
+import org.kie.server.api.model.instance.ErrorInfoInstance;
+import org.kie.server.api.model.instance.RequestInfoInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestSummaryHelper {
```
Comment (cristianonicolai) [inline jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-backend/src/main/java/org/jbpm/console/ng/es/backend/server/RequestSummaryHelper.java]: This class is probably not needed any more as all the mapping is done via *Mapper classes now.

Comment (mswiderski) [inline jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-backend/src/main/java/org/jbpm/console/ng/es/backend/server/RequestSummaryHelper.java]: agreed, will remove this class then

**Comment by mswiderski** [comment]
RequestSummaryHelper has been deleted

**Pull request merged**